cmds/mount/mount.go

```diff
@@ -25,14 +27,38 @@ var (
 	opt    = flag.String("o", "", "Specify mount options")
 )
 
+func informIfUnknownFS(originFS string) {
+	file, err := os.Open("/proc/filesystems")
+	if err != nil {
+		// just don't make things even worse...
+		return
+	}
+	defer file.Close()
+	scanner := bufio.NewScanner(file)
+	for scanner.Scan() {
+		line := scanner.Text()
+		tokens := bufio.NewScanner(strings.NewReader(line))
+		tokens.Split(bufio.ScanWords)
+		var fs string
+		for tokens.Scan() {
+			fs = tokens.Text()
+		}
+		// just check the last token of the line
+		if fs == originFS {
+			return
+		}
+	}
+	log.Printf("Hint: unknown filesystem %s. Check /proc/filesystems for supported ones.\n", originFS)
```

Comment:
well, if you are going to print something, why not print the list of supported file systems that you just accumulated? This is helpful in two ways: it tells me what's supported, and if that ends up being wrong due to some strange bug, I'll know what to blame :-)